src/core/ui.js

```diff
@@ -131,6 +138,31 @@ function createWarnButton(butName, arr, title) {
   return button;
 }
 
+menu.addEventListener("keydown", e => {
+  if (e.key === "Escape") {
+    e.preventDefault();
+    menu.classList.add("respec-hidden");
+    menu.classList.remove("respec-visible");
+    respecPill.setAttribute("aria-expanded", String(menu.hidden));
+    menu.hidden = true;
+    respecPill.focus();
+  } else if (
+    e.target == menu.querySelector("li:first-child").firstElementChild &&
```

Comment:
This can be even shorter by `li:first-child *` without firstElementChild 👍

Reply:
The event.target is that button inside li tags. So the condition won't be true in without using firstElementChild. May be, we have a better alternative to firstElementChild.

Reply:
`li:first-child *` selects the first element child inside of first `li`, so it should be equivalent here.

Reply:
I think that `li:first-child` select the first li which is inside any parent. 
But I will check it again 
Thanks for guidance

Reply:
Note the trailing ` *`, it selects any first element child of the parent (in this case its the first li).